.github/workflows/nightly_dashboard.yml

```
@@ -50,7 +50,7 @@ jobs:
 
       - uses: r-lib/actions/setup-r@v2
         with:
-          r-version: '4.4.0'
+          r-version: '4.5.0'

```

I think this is a nice feature saving a future need to manually bump this. 
```suggestion
          r-version: 'renv'
```